__tests__/cmds/login.test.js

```diff
@@ -11,87 +16,69 @@ describe('rdme login', () => {
-    const password = '123456';
-    const project = 'subdomain';
+    expect(configStore.get('apiKey')).toBe(apiKey);
+    expect(configStore.get('email')).toBe(email);
+    expect(configStore.get('project')).toBe(project);
+    configStore.clear();
+  });
 
-    const mock = nock(config.host).post('/api/v1/login', { email, password, project }).reply(401, {
+  it('should error if invalid credentials are given', async () => {
+    const errorResponse = {
       error: 'LOGIN_INVALID',
       message: 'Either your email address or password is incorrect',
       suggestion: 'You can reset your password at https://dash.readme.com/forgot',
       help: 'If you need help, email support@readme.io and mention log "fake-metrics-uuid".',
-    });
+    };
 
-    return cmd.run({ email, password, project }).catch(err => {
-      expect(err.code).toBe('LOGIN_INVALID');
-      expect(err.message).toContain('Either your email address or password is incorrect');
-      mock.done();
-    });
-  });
+    const mock = nock(config.get('host')).post('/api/v1/login', { email, password, project }).reply(401, errorResponse);
 
-  it('should error if missing two factor token', () => {
-    expect.assertions(2);
-    const email = 'dom@readme.io';
-    const password = '123456';
-    const project = 'subdomain';
+    await expect(cmd.run({ email, password, project })).rejects.toThrow(new APIError(errorResponse));
+    mock.done();
+  });
 
-    const mock = nock(config.host).post('/api/v1/login', { email, password, project }).reply(401, {
+  it('should error if missing two factor token', async () => {
+    const errorResponse = {
       error: 'LOGIN_TWOFACTOR',
       message: 'You must provide a two-factor code',
       suggestion: 'You can do it via the API using `token`, or via the CLI using `rdme login --2fa`',
       help: 'If you need help, email support@readme.io and mention log "fake-metrics-uuid".',
-    });
+    };
+
+    const mock = nock(config.get('host')).post('/api/v1/login', { email, password, project }).reply(401, errorResponse);
 
-    return cmd.run({ email, password, project }).catch(err => {
-      mock.done();
-      expect(err.code).toBe('LOGIN_TWOFACTOR');
-      expect(err.message).toContain('You must provide a two-factor code');
-    });
+    await expect(cmd.run({ email, password, project })).rejects.toThrow(new APIError(errorResponse));
+    mock.done();
   });
 
-  it('should send 2fa token if provided', () => {
-    const email = 'dom@readme.io';
-    const password = '123456';
-    const project = 'subdomain';
+  it('should send 2fa token if provided', async () => {
     const token = '123456';
 
-    const mock = nock(config.host)
+    const mock = nock(config.get('host'))
       .post('/api/v1/login', { email, password, project, token })
       .reply(200, { apiKey: '123' });
 
-    return cmd.run({ email, password, project, token }).then(() => {
-      mock.done();
-    });
+    await cmd.run({ email, password, project, token });
```

Comment:
Should we check the resolved value here for the command output? https://github.com/readmeio/rdme/blob/54151a9f4ee17d4a74bac2fafd9d64d8c49a78ff/src/cmds/login.js#L74